ethereum/contracts/TablelandRigs.sol

```diff
@@ -352,6 +369,30 @@ contract TablelandRigs is
         _unpause();
     }
 
+    // =============================
+    //      PARKING ADMIN LOGIC
+    // =============================
+
+    /**
+     * @dev Throws if called by any account other than parking admin.
+     */
+    modifier onlyAdmin() {
+        _checkAdmin();
+        _;
+    }
+
+    /**
+     * @dev Throws if the sender is not the admin or if admin has not
+     * been initialized.
+     */
+    function _checkAdmin() internal view virtual {
```

Comment:
Looks like this could be `private view`. It doesn't need to be overridden by some inheriting contract like in `Ownable`.